app/controllers/api/trainees/award_recommendations_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Api
+  module Trainees
+    class AwardRecommendationsController < Api::BaseController
+      include Api::Serializable
+
+      def create
+        success, errors = AwardRecommendationService.call(award_recommendation_params, trainee)
```

Comment:
Feels a bit weird service pattern we adopting

Reply:
Feel free to dm me with any questions or suggestions